### PR TITLE
[doc]: Update Command-Reference.md

### PR DESCRIPTION
**Pull request by mdanish-kh**
#### What I did
- Fixed bad hyperlinks from Table of Contents
- Fixed outdated URLs
- Formatting fixes
- Update Azure URLs to sonic-net
- Normalize whitespaces (trim trailing whitespaces)

### CODE REVIEW

**Comment by mdanish-kh** [comment]
@Blueve - Could you please review?

File: doc/Command-Reference.md [hunk 1]
```diff
@@ -10364,44 +10364,85 @@ This command is used to install a new image on the alternate image partition.  T
...
-  Log file system already exists. Size: 4096MB
-  Installed SONiC base image SONiC-OS successfully
-
-  Command: cp /etc/sonic/minigraph.xml /host/
 
-  Command: grub-set-default --boot-directory=/host 0
-
-  Done
-  ```
+```bash
```
Comment (Blueve) [inline doc/Command-Reference.md:10368]: Need add 2 spaces to align with `- Example:`
image

**Review by Blueve — CHANGES_REQUESTED** [review]
Thanks for contribution the doc! Add a comments, could you check it?

File: doc/Command-Reference.md (hunk 1 above)
Comment (mdanish-kh) [reply doc/Command-Reference.md]: Thanks for the suggestion! Done.

**Comment by mdanish-kh** [comment]
Known issue: Kubernetes config commands are missing for which an issue has been created.
- #2443

**Comment by mdanish-kh** [comment]
Known issue: Kubernetes config commands are missing for which an issue has been created.
- #2443

**Comment by mdanish-kh** [comment]
Known issue: Kubernetes config commands are missing for which an issue has been created.
- #2443

**Comment by mdanish-kh** [comment]
@Blueve Could you kindly review again?

**Review by Blueve — APPROVED** [review]

**Comment by Blueve** [comment]
Thank you so much for the contribution! @mdanish-kh

**Comment by mdanish-kh** [comment]
> Thank you so much for the contribution! @mdanish-kh

You're welcome! As always, thanks for quick review!

**Pull request merged**